src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -139,9 +139,10 @@ trait MarathonConf
   }
 
   lazy val groupRoleBehavior = opt[GroupRoleBehavior](
-    name = "group_role_behavior",
-    descr = "Defines the behavior for group roles. 'Top' means that all new child apps and pods are launched" +
-      "with a role that matches the top-level group's name.",
+    name = "new_group_enforce_role",
+    descr = "Used to control whether role enforcement is enabled for new top-level groups. 'Top' means that all new" +
```

Comment:
```suggestion
    descr = "Used to control whether role enforcement is enabled for new top-level groups. 'top' means that all new" +
```

I think it would be consistent to document `top` instead of `Top`. The value is lower cased anyways, and all other mentions here refer to a lower case `top`.

Reply:
Great point